canonical-web-and-design/ubuntu.com
# Copy change; Added 20.04 to bullet points

## Done

- Copy change; Added 20.04 to bullet points

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- [Check against the copy doc and resolve correct changes](https://docs.google.com/document/d/1CiF-dPqG1Y--h4FWfgKB1ft4G3nC2Boa8NT_JG7D6dQ/edit#).


## Issue / Card

Fixes #9530
